### PR TITLE
W-12536520-changes-release-fa

### PR DESCRIPTION
**Pull request by failup**

### CODE REVIEW

**Review by ale-otero-ms — APPROVED** [review]

File: modules/ROOT/pages/_partials/account.adoc [hunk 1]
```diff
@@ -50,11 +50,11 @@ This command accepts the default flag `--help`.
...
 
 ----
-> account:business-group:describe  [flags] <name>
+> account:business-group:describe  [flags] <organizationID>
 ----
 
-This command displays information on the business group you pass in `<name>`. +
-If `<name>` is not specified, the command describes the business group on the current session.
+This command displays information on the business group you pass in `<organizationID>`. +
+If `<organizationID>` is not specified, the command describes the business group on the current session.
```
Comment (ale-otero-ms) [inline modules/ROOT/pages/_partials/account.adoc:57]:
```suggestion
This command displays information on the business group you pass in `<identifier>`. +
If `<identifier>` is not specified, the command describes the business group on the current session.
Identifier searches for id match first, then name.
```

File: modules/ROOT/pages/_partials/account.adoc [hunk 2]
```diff
@@ -50,11 +50,11 @@ This command accepts the default flag `--help`.
 == account:business-group:describe
 
 ----
-> account:business-group:describe  [flags] <name>
+> account:business-group:describe  [flags] <organizationID>
```
Comment (ale-otero-ms) [inline modules/ROOT/pages/_partials/account.adoc:53]:
```suggestion
> account:business-group:describe  [flags] <identifier>
```

**Review by ale-otero-ms — DISMISSED** [review]

**Review by rsuarezmule — DISMISSED** [review]
LGTM!

File: modules/ROOT/pages/_partials/account.adoc [hunk 3]
```diff
@@ -50,11 +50,12 @@ This command accepts the default flag `--help`.
...
 ----
-> account:business-group:describe  [flags] <name>
+> account:business-group:describe  [flags] <identifier>
 ----
 
-This command displays information on the business group you pass in `<name>`. +
-If `<name>` is not specified, the command describes the business group on the current session.
+This command displays information on the business group you pass in `<identifier>`. +
+If `<identifier>` is not specified, the command describes the business group on the current session.
+Identifier searches for id match first, then name.
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/_partials/account.adoc:58]: As a standard we use ID rather than id

File: modules/ROOT/pages/diff-earlier-ver.adoc [hunk 4]
```diff
@@ -13,7 +13,7 @@ Anypoint CLI 4.x introduces the following changes:
 * *Authentication* 
 ** Instead of using the credentials file located in `~/.config/@mulesoft/anypoint-cli-v4-nodejs/config.json`, you set credentials using the `anypoint-cli-v4 conf` command. +
 ** SSO is no longer supported. As a part of Multi-factor authentication (MFA) enablement for all users, the only way to use Anypoint CLI is through xref:access-management::connected-apps-overview.adoc[Connected Apps].
-* *Command Syntax* The separator between command parameters is a colon (:) instead of a space. For example, the command to get the Exchange asset resource list is now `anypoint-cli-v4 exchange:asset:resource:list`.
+* *Command Syntax* The separator between command parameters can also be a colon (:) instead of a space. For example, the command to get the Exchange asset resource list is now `anypoint-cli-v4 exchange:asset:resource:list`.
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/diff-earlier-ver.adoc:16]: Suggestion: 
The separator between command parameters can be either a colon (:) or a space. For example, the command to get the Exchange asset resource list can be either `anypoint-cli-v4 exchange:asset:resource:list` or `anypoint-cli-v4 exchange asset resource list`.

**Review by bjoson-mulesoft — APPROVED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**